### ISSUE TITLE
The OLD IDE is better for me 

### ISSUE THREAD

**Issue by Cyber24BD** [cursor/cursor#2444]
Hi, I like old version, old AI chat handler and old one. 
After updating the latest version there are so many problem in the artificial intelligence communication. I can't talk about certain code with my AI. After selecting some code it will automatically create a new chat. 

There are also so much problem I am facing right now. If backclick sometimes it will be automatically deleted my code base file. 

I really like the old one.

**Comment by OsamaRazaAnsari** [comment]
yes i agree

**Comment by fisforfaheem** [comment]
i cant even udpate :(

**Comment by elvniv** [comment]
i hit upgrade just last night after saying "wow cursor is really good" just to go in today and feel incredibly disappointed. its like it got 40% worse

**Comment by Cyber24BD** [comment]
> i hit upgrade just last night after saying "wow cursor is really good" just to go in today and feel incredibly disappointed. its like it got 40% worse

Yes, I really want to inform the developer that, Code AI Assistant is much better in the oldest version. But we didn't even downgrade our IDE.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
